src/signals.cc

```diff
@@ -130,7 +135,11 @@ NAN_METHOD(DecodeSignal)
         retval = Nan::New((u_int32_t)val);
     }
 
-    info.GetReturnValue().Set(retval);
+    Local<Array> raw_values = Nan::New<v8::Array>(2);
+    Nan::Set(raw_values, 0, retval);
+    Nan::Set(raw_values, 1, Nan::New((u_int32_t) (val >> 32)));
+
+    info.GetReturnValue().Set(raw_values);
```

Comment:
Please confirm my understanding, the issue was that we cannot return 64bit at all as a single value in NodeJS?